pkg/sources/openshift.com/mirror/mirror_test.go

```
@@ -0,0 +1,33 @@
+	}
+
+	if s.Source == nil {
+		t.Fatalf("NewSource() returned a Source with a nil underlying Source")
+	}
+}
+
+// TestBaseURLUsesHTTPS is a security regression test that ensures the mirror.
+// endpoint always uses HTTPS instead of an insecure HTTP connection.
+func TestBaseURLUsesHTTPS(t *testing.T) {
+	parsedURL, err := url.Parse(baseURL)
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofumpt)